functions/java-jre.bats

```diff
@@ -7,4 +7,13 @@ load helpers
   run java -version
   [ "$status" -eq 0 ]
   [[ $output == *"Zulu"* ]]
-}
\ No newline at end of file
+  echo -e "# \e[32mInstallation of Zulu java verified." >&3 
+}
+
+
+@test "installation-java_new" {
+  run java_zulu
+  [ "$status" -eq 0 ]
+  [[ $output == *"Zulu"* ]]
```

Comment:
Why are you checking against "zulu", make no sense?

Reply:
removed